Core/src/main/java/org/tribuo/hash/HashedFeatureMap.java

```diff
@@ -31,16 +39,49 @@
  * does not contain feature name information, but still works
  * with unhashed features names.
```

Comment:
If it will affect the deserialized object, it should document that the underlying hasher will not deserialize properly.

Reply:
Done.